packages/twenty-front/src/modules/settings/data-model/objects/SettingsObjectCoverImage.tsx

```diff
@@ -24,45 +21,25 @@ const StyledCoverImageContainer = styled(Card)`
   height: 153px;
   justify-content: center;
   position: relative;
+  margin-bottom: ${({ theme }) => theme.spacing(8)};
 `;
 
-const StyledTitle = styled.span`
-  color: ${({ theme }) => theme.font.color.tertiary};
-  font-size: ${({ theme }) => theme.font.size.sm};
-  font-weight: ${({ theme }) => theme.font.weight.medium};
+const StyledButtonContainer = styled.div`
   padding-top: ${({ theme }) => theme.spacing(5)};
 `;
-
-const StyledLighIconButton = styled(LightIconButton)`
-  position: absolute;
-  right: ${({ theme }) => theme.spacing(1)};
-  top: ${({ theme }) => theme.spacing(1)};
-`;
-
 export const SettingsObjectCoverImage = () => {
-  const theme = useTheme();
-
-  const [cookieState, setCookieState] = useState(
-    cookieStorage.getItem('settings-object-cover-image'),
-  );
+  const navigate = useNavigate();
 
   return (
-    <AnimatedFadeOut
-      isOpen={cookieState !== 'closed'}
-      marginBottom={theme.spacing(8)}
-    >
-      <StyledCoverImageContainer>
-        <StyledTitle>Build your business logic</StyledTitle>
-        <StyledLighIconButton
-          Icon={IconX}
-          accent="tertiary"
+    <StyledCoverImageContainer>
+      <StyledButtonContainer>
+        <Button
+          Icon={IconEye}
+          title="Visualize"
           size="small"
-          onClick={() => {
-            cookieStorage.setItem('settings-object-cover-image', 'closed');
-            setCookieState('closed');
-          }}
-        />
-      </StyledCoverImageContainer>
-    </AnimatedFadeOut>
+          onClick={() => navigate('/settings/objects/overview')}
```

Comment:
Sorry this change is recent but I just replaced `navigate` by proper `Link`s when it was possible across the app. It's better for accessibility/allows opening in a new tab.